### DISCUSSION TITLE
onOpen/Close for ComboBox, Select and others

### DISCUSSION THREAD

**Discussion by alissaVrk** [tailwindlabs/headlessui#1396] [Ideas]
I like it that I can not manage the open/close state of the dropdown part.
But it will be nice to have an onOpenChange or something on the component.

in my case, I have a combobox with the input inside the options, cause it should disappear with the options. and I'd like to clear the query when the options are hidden

**Comment by silversonicaxel** [comment]
In ComboBox `onClose` exists. `onOpen` not yet.
